WordPress/Classes/ViewRelated/Post/AztecPostViewController.swift

```diff
@@ -2338,6 +2338,7 @@ class MediaProgressCoordinator: NSObject {
         if let mediaUploadingProgress = self.mediaUploadingProgress, !isRunning {
             mediaUploadingProgress.removeObserver(self, forKeyPath: #keyPath(Progress.fractionCompleted))
             self.mediaUploadingProgress = nil
+            self.mediaUploading.removeAll()
```

Comment:
Ah, okay. So can we do this manually from the media library before calling `track...` to start a new set of uploads?